codec/src/main/java/io/netty/handler/codec/protobuf/ProtobufDecoder.java

```diff
@@ -89,10 +89,6 @@ public ProtobufDecoder(MessageLite prototype) {
         this(prototype, null);
     }
 
-    public ProtobufDecoder(MessageLite prototype, ExtensionRegistry extensionRegistry) {
```

Comment:
@AlexanderShniperson we need to keep the constructor as it was public and users may use it in their project. This is a breaking change